cli/install/install.go

```diff
@@ -857,7 +924,26 @@ func installTarantool(binDir string, incDir string, installCtx InstallCtx,
 		}
 
 		if !versionFound {
-			return fmt.Errorf("%s version of tarantool doesn't exist", tarVersion)
+			if len(tarVersion) < 7 {
+				return fmt.Errorf("the hash must include at least 7 characters")
```

Comment:
the hash must `contain` at least 7 characters

Reply:
fixed